c7n/policy.py

```diff
@@ -799,3 +834,27 @@ def get_resource_manager(self):
             raise ValueError(
                 "Invalid resource type: %s" % resource_type)
         return factory(self.ctx, self.data)
+
+    def validate_policy_dt_parse(self):
```

Comment:
last minor, re naming dt_parse is a bit generic, how about validate_start_stop to be more explicit on usage. dt as an abbreviation for datetime may be embedded in various policy filters.